client/src/containers/CategoryForm.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+class CategoryForm extends Component {
+  renderList() {
+    return this.props.categories.map((category) => {
+      return <li key={category.name}>
+      <div className="fl w-50 w-50-ns pa2 ">
+        <button className="hover-bg-orange bg-white pv4 h4 w4 br4">
+          <label className="f4 black">{category.name}</label>
+          <br />
+          <img
+            alt=""
```

Comment:
let's add a text here for the sake of accessibility 

PS: almost no one cares about accessibility :laughing: in the tech industry